### PR TITLE
Feature/itc 104 add link to create header 11 2020

### PR DESCRIPTION
**Pull request by sinejespersen**
https://jira.itkdev.dk/browse/ITC-104

### CODE REVIEW

File: html/modules/custom/itchefer_link_in_create_header/itchefer_link_in_create_header.module [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * The module for itchefer_event_enrollment.
```
Comment (rimi-itk) [inline html/modules/custom/itchefer_link_in_create_header/itchefer_link_in_create_header.module]: Hide your tracks when you steal code.

File: html/modules/custom/itchefer_link_in_create_header/itchefer_link_in_create_header.module [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ */
+
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\Core\Url;
+
+/**
+ * @file
+ * The Social event module.
```
Comment (rimi-itk) [inline html/modules/custom/itchefer_link_in_create_header/itchefer_link_in_create_header.module]: Well, I'm not sure this description is accurate …

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Some comments should be fixed.

File: html/modules/custom/itchefer_link_in_create_header/itchefer_link_in_create_header.module (hunk 1 above)
Comment (sinejespersen) [reply html/modules/custom/itchefer_link_in_create_header/itchefer_link_in_create_header.module]: haha hvor pinligt

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**